### DISCUSSION TITLE
Make it so it works on static websites like cloudflare, netifly, vercel, and etc

### DISCUSSION THREAD

**Discussion by No123567** [NebulaServices/Nebula#234] [Ideas]
a

**Comment by diamondbroplayz** [comment]
Really, that's not possible without an outside bare server. If you'd like, modify the SW and add a service worker (such as `https://tomp.app/`)

**Reply by No123567** [reply]
Ok thanks

**Reply by 88411** [reply]
Can you explain how to do that?

**Reply by diamondbroplayz** [reply]
i’m pretty sure you go find uv.config.js and modify the bare in there, but I haven’t messed with proxies in a while
